### PR TITLE
Improvement to Admin Documentation of upgrades: show examples of CDAP version

### PR DESCRIPTION
**Pull request by awholegunch**
Add version examples, and make current examples independent of the version.
Adds reST "replacements" for the "short version", "previous short version", and bold and literal versions of each.

Passes [Quick Build 1](http://builds.cask.co/browse/CDAP-DOB142-1).

Pages of Interest:
- [Upgrading CDAP with CM](http://builds.cask.co/artifact/CDAP-DOB142/shared/build-1/Docs-HTML/3.4.0-SNAPSHOT/en/admin-manual/upgrading/cloudera.html#admin-upgrading-cloudera)
- [Upgrading CDAP using Packages](http://builds.cask.co/artifact/CDAP-DOB142/shared/build-1/Docs-HTML/3.4.0-SNAPSHOT/en/admin-manual/upgrading/packages.html)

### CODE REVIEW

File: cdap-docs/admin-manual/source/upgrading/cloudera.rst [hunk 1]
```diff
@@ -16,6 +16,8 @@ Upgrading CDAP
 
 Upgrading CDAP Patch Release Versions
 -------------------------------------
+Upgrading between patch versions of CDAP refers to upgrading from one |short-version|\.x
+version to a higher |short-version| version.
```
Comment (dereklwood) [inline cdap-docs/admin-manual/source/upgrading/cloudera.rst]: suggestion, if you agree:  to a higher |short-version|**\\\.x** version

File: cdap-docs/admin-manual/source/upgrading/cloudera.rst [hunk 2]
```diff
@@ -30,7 +32,8 @@ in the Cloudera Manager UI.
 
 Upgrading CDAP Major/Minor Release Versions
 -------------------------------------------
-Upgrading between major versions of CDAP involves the additional steps of upgrading the
+Upgrading between major versions of CDAP (for example, from a 3.3.x version to |short-version|\.x)
```
Comment (dereklwood) [inline cdap-docs/admin-manual/source/upgrading/cloudera.rst]: instead of hardcoding ``3.3.x``, can we use your ``|previous-short-version|\.x`` here?

**Comment by dereklwood** [comment]
Couple minor suggestions, at your discretion.  :+1: LGTM

File: cdap-docs/admin-manual/source/upgrading/cloudera.rst (hunk 1 above)
Comment (awholegunch) [inline cdap-docs/admin-manual/source/upgrading/cloudera.rst]: Fixed.

File: cdap-docs/admin-manual/source/upgrading/cloudera.rst (hunk 2 above)
Comment (awholegunch) [inline cdap-docs/admin-manual/source/upgrading/cloudera.rst]: Good catch. Fixed.

**Pull request merged**